### ISSUE TITLE
Swapfile/pagefile getting maxed out. Using equivalent of 65GB of RAM!

### ISSUE THREAD

**Issue by andy8992** [lllyasviel/stable-diffusion-webui-forge#2370]
I noticed that my pagefile is being redlined taking it up to the max eventually. Any idea why this is happening?

Surely SDXL doesn't take 70GB of RAM to run..

**Comment by MisterChief95** [comment]
Can you provide this information:

- GPU make/model/vram amount
- System RAM amount
- `GPU Weights (MB)` setting at the top of the UI
    ![image](https://github.com/user-attachments/assets/78e37539-a383-4a90-8e4c-998dbd36b3d4)
    <sub>(this setting)</sub>

**Comment by andy8992** [comment]
4060 Laptop 8GB Vram
32GB Sys Ram
6000mb usually

**Comment by MisterChief95** [comment]
Yeah, I agree. I don't think your system should be falling back to swap like that. When using SDXL does your system ram get maxed out? I'm surprised its opting to use swap since 32GB RAM should definitely be enough to handle SDXL + Hires Fix.

Anyway, I did some testing and here are my main suggestions:
1. Tweak your GPU Weights to be a bit lower, around 4,000-5,000MB. I'm wondering if 6,000MB isn't leaving enough VRAM for inference to occur since we have to account for the fact that Windows itself reserves at least ~1GB of VRAM.
2. If you switch to the `all` radio button on the top left of the UI, see if enabling `Async` Swap Method, and `Shared` Swap Location options help at all. That will offload to the shared GPU memory.
3. All else fails, you can try setting the `Diffusion in low bits` option to one of the float8 types. That should reduce the model size enough to fit the whole thing in VRAM and have enough left over for inference.
